src/linkplay/utils.py

```diff
@@ -67,6 +75,50 @@ async def session_call_api_ok(
         raise LinkPlayRequestException(f"Didn't receive expected OK from {endpoint}")
 
 
+async def call_tcpuart(host: str, cmd: str) -> str | None:
```

Comment:
This is also no `async` function, as the connecting/sending bytes happens synchronously